website/docs/security/security-hardening.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Hardening authentik
+---
+
+While authentik is secure out of the box, you can take steps to further increase the security of an authentik instance. As everyone knows, there is a consequential tradeoff between security and convenience. All of these hardening practices have an impact on the user experience and should only be applied knowing this tradeoff.
+
+### Expressions
+
+[Expressions](../property-mappings/expression.mdx) allow super-users and other highly privileged users to create custom logic within authentik to modify its behaviour. Editing/creating these expressions is, by default, limited to super-users and any related events are fully logged.
+
+However, for further hardening, it is possible to prevent any user (even super-users) from using expressions to create or edit any objects. To do so, configure your deployment to block API requests to these endpoints:
+
+-   `/api/v3/policies/expression*`
+-   `/api/v3/propertymappings*`
+-   `/api/v3/managed/blueprints*`
+
+With these restrictions in place, expressions can only be edited using [Blueprints on the file system](https://docs.goauthentik.io/developer-docs/blueprints/#storage---file). And of course, implement restricted access to the file system itself.
```

Comment:
Suggestion: "Take care to restrict access to the file system itself."  That "... and of course" seems a bit off-hand.